docs/client/python/pyhive.md

```diff
@@ -0,0 +1,70 @@
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+
+# PyHive
+
+[PyHive](https://github.com/dropbox/PyHive) is a collection of Python DB-API and SQLAlchemy interfaces for Hive. PyHive is able to connect with Kyuubi server as Kyuubi server supports serving in thrift protocol as HiveServer2.
+
+PyHive is now marked as [no-longer maintained](https://github.com/dropbox/PyHive/commit/d199a1bd55c656b5c28d0d62f2d3f2e6c9a82a54) and staying on HiveServer2 Thrift bindings v11 which is compatible with Kyuubi server.
+
+## Requirements
+PyHive works with Python 2.7 / Python 3. Install PyHive via pip for the Hive interface.
+
+```
+pip install 'pyhive[hive]'
+```
+
+## Usage
+Use Kyuubi server's host and thrift protocol port to connect with.
```

Comment:
Use **the** Kyuubi server's host and thrift protocol port to connect ~~with~~.

Reply:
ok, applied.